### ISSUE TITLE
Can not get Facebook images to work with share component.

### ISSUE THREAD

**Issue by monstrfolk** [MurhafSousli/ngx-sharebuttons#131]
`<share-buttons
            class="hidden-xs btn"
            [reddit]="false"
            [pinterest]="false"
            [linkedIn]="false"
            [tumblr]="false"
            [stumbleUpOn]="false"
            [url]="shareService.link | async"
            [image]="shareService.image | async"
            [description]="shareService.description | async"></share-buttons>`

The url generated with the share button for Facebook.

https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fqa.musehoso.com%2Fb-g%2Fview%2Fportfolio%2FFaFJE9Y8ktjuZg8AD%2FAtfTC97DwLW7QFMeP

Title, link, and description work.

What am i doing wrong?

**Comment by MurhafSousli** [comment]
@monstrfolk  First of all, your website is offline. **facebook** cannot access your website to get the info.
also you won't be able to test it while developing in a local environment for the same reason.

_You can test it when your app is up and running, use facebook share debugger for that
https://developers.facebook.com/tools/debug/sharing/?q=https://musehoso.com_

Secondly, there is a problem with your `shareService`, the async pipe is not working, as you see the share link is not adding `image` and `description` parameters

**Comment by monstrfolk** [comment]
The url for the website is not correct. I changed that on purpose. 

Below is the link generate from the live example (https://murhafsousli.github.io/ngx-sharebuttons/). There is no image or description parameters but still works.

https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fmurhafsousli.github.io%2Fngx-sharebuttons%2F&description=Simple,%20lightweight,%20customizable%20share%20buttons%20with%20counts

**Comment by MurhafSousli** [comment]
@monstrfolk The demo is SEO friendly it uses the meta tags, it doesn't need these inputs.
Yet I used them in different places to demonstrate different use cases. 

for example the link you just pasted in your comment, uses the `description` input, have another look at it `&description=Simple,%20lightweight,%20customizable%20share%20buttons%20with%20counts`
 
In another example I used it `image` input for **Pinterest** button for a custom image.

Use the meta tags, The inputs are only alternates for them.
